scripts/libunwind-container-support.patch

```diff
@@ -0,0 +1,74 @@
+commit 831459ee961e7d673bbd83e40d0823227c66db33
```

Comment:
Did you make any changes over it? If so, which? (comment in code maybe), and if not, can replace with `curl https://github.com/../../...patch`

Reply:
I did

Reply:
Actually, I can make this changes with a simple `sed`

Reply:
kk